### ISSUE TITLE
prometheus-kube-stack issue/warnings deploying on cluster created  with minikube and kind

### ISSUE THREAD

**Issue by mudrii** [prometheus-community/helm-charts#345]
**Describe the bug**
Multiple warnings during the installation of the Prometheus helm chart on minikube and kind
```
minikube version
minikube version: v1.14.2
commit: v1.14.2

kind version
kind v0.9.0 go1.15.4 linux/amd64
```
Warnings on Minikube
```
helm install test prometheus-community/kube-prometheus-stack
W1112 01:52:27.353133  815669 warnings.go:67] rbac.authorization.k8s.io/v1beta1 Role is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 Role
W1112 01:52:27.356698  815669 warnings.go:67] rbac.authorization.k8s.io/v1beta1 RoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 RoleBinding
W1112 01:52:39.149249  815669 warnings.go:67] rbac.authorization.k8s.io/v1beta1 Role is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 Role
W1112 01:52:39.240182  815669 warnings.go:67] rbac.authorization.k8s.io/v1beta1 RoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 RoleBinding
NAME: test
LAST DEPLOYED: Thu Nov 12 01:52:26 2020
NAMESPACE: default
STATUS: deployed
REVISION: 1
NOTES:
kube-prometheus-stack has been installed. Check its status by running:
  kubectl --namespace default get pods -l "release=test"

Visit https://github.com/prometheus-operator/kube-prometheus for instructions on how to create & configure Alertmanager and Prometheus instances using the Operator.
```

Warnings on kind

```
helm install test prometheus-community/kube-prometheus-stack                                      344ms
W1112 02:05:18.395131  837681 warnings.go:67] rbac.authorization.k8s.io/v1beta1 Role is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 Role
W1112 02:05:18.397336  837681 warnings.go:67] rbac.authorization.k8s.io/v1beta1 RoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 RoleBinding
W1112 02:05:30.624143  837681 warnings.go:67] rbac.authorization.k8s.io/v1beta1 Role is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 Role
W1112 02:05:30.627127  837681 warnings.go:67] rbac.authorization.k8s.io/v1beta1 RoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 RoleBinding
NAME: test
LAST DEPLOYED: Thu Nov 12 02:05:17 2020
NAMESPACE: default
STATUS: deployed
REVISION: 1
NOTES:
kube-prometheus-stack has been installed. Check its status by running:
  kubectl --namespace default get pods -l "release=test"

Visit https://github.com/prometheus-operator/kube-prometheus for instructions on how to create & configure Alertmanager and Prometheus instances using the Operator.
```

**Version of Helm and Kubernetes**:

Helm Version:

```console
$ helm version
version.BuildInfo{Version:"v3.4.0", GitCommit:"", GitTreeState:"", GoVersion:"go1.15.4"}
```

Kubernetes Version: minikube

```console
$ kubectl version --short
Client Version: v1.19.3
Server Version: v1.19.2
```
Kubernetes Version: kind

```
kubectl version --short
Client Version: v1.19.3
Server Version: v1.19.1
```

**Which chart**:
**Which version of the chart**:
Latest

**What happened**:
warning during installation with helm

**What you expected to happen**:
Both cases installation succeded

**How to reproduce it** (as minimally and precisely as possible):
Deploy clusters with minikube and kind

**Comment by monotek** [comment]
Install a more recent minikube version.ö
Kubernetes 1.14 is eol for some time already...

**Comment by mudrii** [comment]
> Install a more recent minikube version.ö
> Kubernetes 1.14 is eol for some time already...

@monotek you confusing Kubernetes version with Minikube version

Kubernetes version is 1.19.1 for minikube and 1.19.2 for kind

as for minikube yes version is 1.14.2 released on 27.10.2020  https://github.com/kubernetes/minikube/blob/master/CHANGELOG.md

**Comment by inductor** [comment]
Same here. Chart should be fixed by K8s 1.22 release.
